src/core/hyper-ref/index.js

```diff
@@ -106,7 +107,12 @@ export default class HyperRef extends PureComponent<Props, State> {
     const action =
       behaviorElement.getAttribute(ATTRIBUTES.ACTION) || NAV_ACTIONS.PUSH;
 
-    if (Object.values(NAV_ACTIONS).indexOf(action) >= 0) {
+    if (action === ACTIONS.RELOAD) {
```

Comment:
wouldn't it be desirable to also support delay and loading indicator like `NAV_ACTIONS` case?

Reply:
it would be, I'll add an Asana task for this:
https://app.asana.com/0/923014529014430/1143270339705747/f